app/helpers/format-email.js

```diff
@@ -13,7 +14,7 @@ export function formatEmail(email) {
         ret = `<a href='mailto:${escape(formatted[2])}'>${ret}</a>`;
     }
 
-    return ret.htmlSafe();
```

Comment:
this is going to be deprecated anyway ;)

Reply:
correct